secureli/modules/core/core_services/scanner.py

```diff
@@ -18,15 +22,7 @@ class OutputParseErrors(str, Enum):
     REPO_NOT_FOUND = "repo-not-found"
 
 
-class ScanOuput(pydantic.BaseModel):
```

Comment:
I moved this to the models folder along with `ScanFailure`, `ScanMode`, and `ScanResult`. It felt more appropriate there.